core/client/fs/src/test/java/alluxio/client/file/cache/CacheManagerTest.java

```diff
@@ -40,4 +62,21 @@ public void factoryClear() throws Exception {
     CacheManager.Factory.clear();
     assertNotEquals(manager, CacheManager.Factory.get(mConf));
   }
-}
+
+  private static class CacheManagerFetch implements Callable<CacheManager> {
+    InstancedConfiguration mConf = null;
+
+    public CacheManagerFetch(InstancedConfiguration mConf) {
+      this.mConf = mConf;
+    }
```

Comment:
Alluxio code convention:

```suggestion
    public CacheManagerFetch(InstancedConfiguration conf) {
      mConf = mConf;
    }
```